### DISCUSSION TITLE
Calcultate feature in the new view feature

### DISCUSSION THREAD

**Discussion by dauphinpasdroit** [Budibase/budibase#12660] [Help]
Hello guys,

I'm looking for the calculate feature in the new view panel to create some specific charte pie like here : [#11093](url)

But the calculate feature just disappeared in the new view panel and I couldn't find a way to do the same as the old one...

Thank you for you help

**Comment by ConorWebb96** [comment]
Hey @alphaxr6,

I'm not quite sure what you mean here, the link you provided hits a 404 page so I wasn't able to see the additional context.

**Reply by dauphinpasdroit** [reply]
Hey @ConorWebb96 

It seems like the link didn't work. Here it what I referred to : https://github.com/Budibase/budibase/discussions/11093

**Reply by ConorWebb96** [reply]
I understand now, the group by functionality you are after was removed for performance reasons. Therefore you won't be able to make the view that was shown in that post. There is a more detailed explanation on this [here](https://github.com/Budibase/budibase/discussions/12211#discussioncomment-7436100).

You may be able to use the Rest data source in combination with the public api to achieve a similar effect that was showcased in the discussion you linked.

I hope this makes sense
